packages/fuels/src/cli/commands/build/buildSwayProgram.ts

```diff
@@ -9,12 +8,11 @@ import { onForcExit, onForcError } from './forcHandlers';
 export const buildSwayProgram = async (config: FuelsConfig, path: string) => {
   debug('Building Sway program', path);
 
-  return new Promise<void>((resolve, reject) => {
-    const builtInForcPath = findBinPath('fuels-forc', __dirname);
+  const forcPath = config.forcPath ?? 'forc';
```

Comment:
It seems the conditional is not needed here, as you did it when [loading the config](https://github.com/FuelLabs/fuels-ts/pull/2143/files#diff-93dc7bf4f6d1b4307daf7c9ff4c0adf3902d71b5809de0137dc2314e0f91db05R46-R47).